torch_xla/csrc/ops/conv2d.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "ir.h"
+#include "tensorflow/compiler/xla/xla_data.pb.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+
+// IR node for 2D convolutions with or without bias.
+class Conv2d : public Node {
+ public:
+  Conv2d(const NodeOperand& input, const NodeOperand& weight,
+         const NodeOperand& bias, int stride, int padding,
+         bool use_full_conv_precision);
+
+  Conv2d(const NodeOperand& input, const NodeOperand& weight, int stride,
+         int padding, bool use_full_conv_precision);
+
+  XlaOpVector Lower(LoweringContext* loctx) const override;
+
+ private:
+  // The parameters of the convolution. Only support the same stride and padding
+  // in both dimension for now.
+  const int stride_;
```

Comment:
I am not totally opposed to const here, but we have a lot of code that does not have them, although constants.
This reminds me the Java's final a bit.
I'd keen to not have those.

Reply:
I've removed them, although I think we should instead fix the rest of the code to have them.